lib/test/structuredfield/decodeId3TextFrame.test.ts

```diff
@@ -0,0 +1,23 @@
+import { equal } from 'node:assert';
+import { describe, it } from 'node:test';
+import { decodeId3TextFrame } from '../../src/id3/util/decodeId3TextFrame.js';
+import { Id3Frame } from '../../src/id3';
+
+describe('decodeId3TextFrame', () => {
+	it('should decode a TXXX frame', () => {
+		const frame = {
+			type: 'TXXX',
+			data: new Uint8Array([0, 102, 111, 111, 0, 97, 98, 99]),
+			size: 2, // required by the _decodeTextFrame function
```

Comment:
This comment comes from Shaka, we should eigther remove the '_' or remove the comment.